spec/lib/reportsv2_spec.rb

```diff
@@ -237,7 +237,7 @@
       end
     end
 
-    context 'XLS reports' do
+    context 'XLS reports', :pro_account do
```

Comment:
Unfortunately original master branch still fails several tests for me like on Travis. The master branch doesn't have 'passed' status there - https://travis-ci.org/kanet77/togglv8/branches